pkg/worker/job.go

```diff
@@ -42,7 +42,7 @@ type Job struct {
 
 	// Execution information.
 	AttemptsDone  int       `json:"attempts_done"`
-	Status        JobStatus `json:"status"`
```

Comment:
why is this edited?

Reply:
`Status` is not persisted to the database so it is not populated when we fetch jobs from it. The status is set to dead or done after job execution as a signal for further handling. So including the field in the JSON response does not add any value.